MAINTAINERS.md

```diff
@@ -27,18 +27,18 @@ If you are an owner of the organization, you should be able to see an automated
   [12 March 2017](https://github.com/tldr-pages/tldr/issues/1209#issuecomment-285924778) — present
 - **Muhammad Falak R Wani ([@mfrw](https://github.com/mfrw))**:
   [7 September 2018](https://github.com/tldr-pages/tldr/issues/2306) — present
-- **Lucas Schneider ([@schneiderl](https://github.com/schneiderl))**:
-  [11 April 2019](https://github.com/tldr-pages/tldr/issues/2898) — present
 - **Andrik Albuquerque ([@andrik](https://github.com/andrik))**:
   [9 May 2019](https://github.com/tldr-pages/tldr/issues/2988) — present
 - **Ivan Aracki ([@Aracki](https://github.com/Aracki))**:
   [9 May 2019](https://github.com/tldr-pages/tldr/issues/2988) — present
-- Ein Verne ([@einverne](https://github.com/einverne)):
-  [29 October 2019](https://github.com/tldr-pages/tldr/issues/3488) — [12 January 2020](https://github.com/tldr-pages/tldr/issues/3738)
 - **Pierre Rudloff ([@Rudloff](https://github.com/Rudloff))**:
   [19 November 2019](https://github.com/tldr-pages/tldr/issues/3580) — present
 - **Proscream ([@Proscream](https://github.com/Proscream))**:
   [22 November 2019](https://github.com/tldr-pages/tldr/issues/3592) — present
+- Lucas Schneider ([@schneiderl](https://github.com/schneiderl)):
+  [11 April 2019](https://github.com/tldr-pages/tldr/issues/2898) — [17 January 2019](https://github.com/tldr-pages/tldr/issues/3764)
+- Ein Verne ([@einverne](https://github.com/einverne)):
+  [29 October 2019](https://github.com/tldr-pages/tldr/issues/3488) — [12 January 2020](https://github.com/tldr-pages/tldr/issues/3738)
 - Zlatan Vasović ([@zdroid](https://github.com/zdroid)):
   [30 November 2019](https://github.com/tldr-pages/tldr/issues/3636) — [17 December 2019](https://github.com/tldr-pages/tldr/issues/3663)
 - Iván  ([@ivanhercaz](https://github.com/ivanhercaz)):
```

Comment:
@zdroid, would it be possible to change my name to my name + surnames in this PR? If you agree, it would be Iván Hernández Cazorla or Iván Hernández-Cazorla.

Reply:
I think we can do this in a separate PR as the changes are not related.

Reply:
@schneiderl, thank you for your opinion! Yes, you have reason, it is better to not mix different changes in the same PR. I will do by myself this change.